lib/functions/optimizers.ts

```diff
@@ -37,4 +53,38 @@ export default function gradientDescent(
     return update
 }
 
-export { gradientDescent, mseCostFunction }
+export function updateWeightsAndBias(
```

Comment:
`optimizers.ts` should not export `updateWeightsAndBias` and `updateWeightAndBias` but they should exist as separate functionalities. I suggest to create a single function that checks the input format and calls either one or the other